Exercise_1.java

```diff
@@ -1,44 +1,75 @@
-class Stack { 
-    static final int MAX = 1000; 
-    int top; 
-    int a[] = new int[MAX]; // Maximum size of Stack 
-  
-    boolean isEmpty() 
-    { 
-        //Write your code here 
-    } 
-
-    Stack() 
-    { 
-        //Initialize your constructor 
-    } 
-  
-    boolean push(int x) 
-    { 
+class Stack1 {
+    static final int MAX = 1000;
+    int top;
+    int a[] = new int[MAX]; // Maximum size of Stack
+
+    boolean isEmpty()
+    {
+        //Write your code here
+        if (top == -1){
+            return true;
+                      }
+        else {
+            return false ;
+             }
+        }
+
+    Stack1()
+    {
+        //Initialize your constructor
+        top = 0;

```

Comment:
Here the stack is initialized to top = 0 but the stack is empty when top = -1. So the stack has no elements when it's initialized but isEmpty will return False.